lms/envs/common.py

```diff
@@ -1637,7 +1637,8 @@
     'js/vendor/moment.min.js',
     'js/vendor/url.min.js',
     'js/courseware/course_home_events.js',
-    'js/courseware/toggle_element_visibility.js'
+    'js/courseware/toggle_element_visibility.js',
+    'js/vendor/draggabilly.pkgd.js',
```

Comment:
As we discussed, you can add the Pattern Library JS files here so that they will be loaded correctly by the video player.